Source/UI/IdentityUI.swift

```diff
@@ -462,6 +468,11 @@ extension IdentityUI {
         case let .byRoute(route, vc):
             handleRouteForUnpresentedUI(route: route, byPresentingIn: vc, client: client, completion: completion)
         case let .byLoginMethod(loginMethod, vc, localizedTeaserText, scopes):
+            if #available(iOS 13.0, *), case .sharedWebCredentials = loginMethod {
```

Comment:
nit: Why do we have to additionally check available version as `sharedWebCredentials` is available as of iOS 13?

Reply:
Because you can't call a iOS 13 method without a #available check. It's required by the compiler.